pkgs/applications/networking/cluster/kubevpn/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "kubevpn";
-  version = "2.0.1";
+  version = "2.1.0";
 
   src = fetchFromGitHub {
     owner  = "KubeNetworks";
     repo   = "kubevpn";
     rev    = "v${version}";
-    sha256 = "sha256-vlUR/fr/PFUBzvxebssrcUgfgnDpFuLuJOl5SBEMKR8=";
+    sha256 = "sha256-TWvXXwEOAxku2+pesk8tSebRnrCwhwfiHpke/c9InWM=";
   };
 
-  vendorHash = "sha256-EcHHSOAr93M2kPhEaZSpvpZCq4pi1NCedo9hw1Mn0a4=";
+  vendorHash = null;
```

Comment:
Why is this null?

Reply:
The robot must have detected that the dependencies are vendored.